gdk-pixbuf/src/subclass/pixbuf_animation.rs

```diff
@@ -32,11 +35,18 @@ pub trait PixbufAnimationImpl: ObjectImpl {
 }
 
 mod sealed {
+    use glib::{prelude::*, subclass::prelude::*};
     pub trait Sealed {}
-    impl<T: super::PixbufAnimationImplExt> Sealed for T {}
+    impl<T: super::PixbufAnimationImplExt> Sealed for T where
+        <T as ObjectSubclass>::Type: IsA<super::PixbufAnimation>
+    {
+    }
 }
 
-pub trait PixbufAnimationImplExt: sealed::Sealed + ObjectSubclass {
+pub trait PixbufAnimationImplExt: sealed::Sealed + ObjectSubclass
```

Comment:
I wonder if the sealing is still necessary like this

Reply:
Also now that I think about it, is `pub trait PixbufAnimationExt: IsA<PixbufAnimation>` required to be sealed? It's implemented over all `IsA<PixbufAnimation>` so there can't really be any other implementation, can there?

Removing the sealing would probably reduce compile times because lot fewer traits?

Reply:
```suggestion
pub trait PixbufAnimationImplExt: sealed::Sealed + ObjectSubclass + PixbufAnimationImpl
```

too maybe?